coq.opam

```diff
@@ -1,4 +1,4 @@
-# This file is generated by dune, edit dune-project instead
+# This file is NOT generated by dune (runtest doesn't work)
```

Comment:
How do I get dune to stop complaining that it wants to own the opam files?

Reply:
You want to add a `coq.opam.template` file.

Note that I don't think you should update this, if Coq was already configured in `coq-core`, you should not reconfigure it here.